src/syrupy/extensions/base.py

```diff
@@ -123,6 +123,12 @@ def write_snapshot(self, *, data: "SerializedData", index: int) -> None:
         """
         self._pre_write(data=data, index=index)
         snapshot_location = self.get_location(index=index)
+        if not self.test_location.matches_snapshot_location(snapshot_location):
+            warning_msg = f"""
+            Can not relate snapshot location '{snapshot_location}' to the test location.
```

Comment:
(nb) i18n gettext

non-blocking since we still haven't done any sort of translations